package.json

```diff
@@ -36,14 +36,17 @@
     "husky": "9.0.11",
     "lint-staged": "15.2.2",
     "tsconfig": "workspace:*",
-    "typescript": "^5.4.5",
-    "vitest": "^1.5.0"
+    "typescript": "^5.6.2",
+    "vitest": "^1.6.0"
   },
   "packageManager": "pnpm@9.1.1",
   "pnpm": {
     "overrides": {
       "@pmmmwh/react-refresh-webpack-plugin": "0.5.11",
       "react-refresh": "0.14.0"
     }
+  },
+  "dependencies": {
+    "parcel": "^2.12.0"
   }
```

Comment:
Please don't make changes to package versions that are unrelated to the main purpose of the pull request.

Reply:
alright sorry will keep in mind!